### PR TITLE
Update camera keyboard tests

### PR DESCRIPTION
**Pull request by mikeger**
- Due to the issue that Simulator must request permissions for our app to access camera roll testing on CI with real camera roll does not work
- Some tests that where relaying on real camera roll are reverted to fix CI
- As next we need to think on how this issue can be avoided

### CODE REVIEW

**Comment by daehn** [comment]
Maybe something like this can help: https://github.com/plu/JPSimulatorHacks?

**Comment by mikeger** [comment]
@daehn thanks, I'll check

File: Wire-iOS Tests/CameraKeyboardViewControllerTests.swift [hunk 1]
```diff
@@ -57,10 +58,12 @@ final class CameraKeyboardViewControllerTests: ZMSnapshotTestCase {
     var sut: CameraKeyboardViewController!
     var splitView: SplitLayoutObservableMock!
     var delegateMock: CameraKeyboardViewControllerDelegateMock!
-    let assetLibrary = AssetLibrary(synchronous: true)
+    var assetLibrary: AssetLibrary!
     
     override func setUp() {
         super.setUp()
+        JPSimulatorHacks.grantAccessToPhotos()
```
Comment (daehn) [inline Wire-iOS Tests/CameraKeyboardViewControllerTests.swift]: 👍

**Comment by mikeger** [comment]
@daehn @typfel we had a bug with mocking `ZMUser` class that caused crash when spinning run loop in tests. Now it is fixed (`JPSimulatorHacks` spins run loop)

**Comment by mikeger** [comment]
Ok bob is still having issues with it, let's wait until tests can be run on Bob

**Pull request merged**